--- parser.cpp	2013-06-30 20:54:41.023253319 +0200
+++ parser.cpp.new	2013-06-30 20:53:32.163251173 +0200
@@ -4678,15 +4678,15 @@ yyreportSyntaxError (yyGLRStack* yystack
   switch (yycount)
     {
 #define YYCASE_(N, S)                   \
-      case N:                           \
-        yyformat = S;                   \
-      break
-      YYCASE_(0, YY_("syntax error"));
-      YYCASE_(1, YY_("syntax error, unexpected %s"));
-      YYCASE_(2, YY_("syntax error, unexpected %s, expecting %s"));
-      YYCASE_(3, YY_("syntax error, unexpected %s, expecting %s or %s"));
-      YYCASE_(4, YY_("syntax error, unexpected %s, expecting %s or %s or %s"));
-      YYCASE_(5, YY_("syntax error, unexpected %s, expecting %s or %s or %s or %s"));
+	case N:                           \
+					  yyformat = S;                   \
+	break
+	YYCASE_(0, YY_("Unknown syntax error"));
+	YYCASE_(1, YY_("Syntax error: unexpected %s"));
+	YYCASE_(2, YY_("Syntax error: unexpected %s\n	expecting %s"));
+	YYCASE_(3, YY_("Syntax error: unexpected %s\n	expecting %s or %s"));
+	YYCASE_(4, YY_("Syntax error: unexpected %s\n	expecting %s, %s or %s"));
+	YYCASE_(5, YY_("Syntax error: unexpected %s\n	expecting %s, %s, %s or %s"));
 #undef YYCASE_
     }
 
